authorization/is_authorized.go

```diff
@@ -77,12 +80,15 @@ func (c *Client) IsAuthorizedByStringExternalID(
 	opts ...grpc.CallOption,
 ) (*authorizationpb.IsAuthorizedResponse, error) {
 	return c.IsAuthorizedWithRawRequest(ctx, &authorizationpb.IsAuthorizedRequest{
-		Subject: &identitypb.DigitalTwinIdentifier{Filter: &identitypb.DigitalTwinIdentifier_Property{
-			Property: &identitypb.Property{
-				Definition: &identitypb.PropertyDefinition{Property: externalIDProperty},
-				Value:      &identitypb.Property_ObjectValue{ObjectValue: objects.String(externalID)},
-			},
-		}},
+		Subject: &authorizationpb.IsAuthorizedRequest_DigitalTwinIdentifier{
+			DigitalTwinIdentifier: &identitypb.DigitalTwinIdentifier{
+				Filter: &identitypb.DigitalTwinIdentifier_PropertyFilter{
+					PropertyFilter: &identitypb.PropertyFilter{
+						Type:  externalIDProperty,
+						Value: objects.String(externalID),
```

Comment:
Is the authorization backend using the new propertyfilter @bakkelim ?

Reply:
Actually was the stuff that is being changed here, even correct? To my knowledge the `DigitalTwinIdentifier_Property` was not implemented and had no effect.